spec/models/book_spec.rb

```diff
@@ -45,4 +46,63 @@
     expect(book).to be_valid
   end
 
+  describe "#author_name" do
+    it "does not return the first name first" do
+      author = FactoryBot.create(:author,
+                                first_name: "Malcom",
+                                last_name:  "Gladwell")
+      book   = FactoryBot.create(:book,
+                                 title: "Blink",
+                                 author: author)
+      result = book.author_name
+
+      expect(result).to_not eq("Malcom, Gladwell")
+    end
+
+    it "returns the author's name in the correct format" do
+      author = FactoryBot.create(:author,
+                                first_name: "Malcom",
+                                last_name:  "Gladwell")
+      book   = FactoryBot.create(:book,
+                                 title: "Blink",
+                                 author: author)
+      result = book.author_name
+
+      expect(result).to eq("Gladwell, Malcom")
+    end
+  end
+
+  describe "#book_format_types" do
+    it "returns a collection of BookFormatTypes for single book" do
+      book  = create(:book)
+      create(:book_format,
+             book: book,
+             book_format_type: create(:book_format_type,
+                                      name: "Hardcover",
+                                      physical: true))
+      create(:book_format,
+             book: book,
+             book_format_type: create(:book_format_type,
+                                      name: "PDF"))
+
+      result = book.book_format_types
```

Comment:
@ryangarver the spec asks that #book_format_types returns a collection of the types for one book - would you prefer to just have the names of each type, or is a collection of BookFormatType instances OK? Thanks for letting me know!